src/main/jte/bookings/single.jte

```diff
@@ -20,12 +20,21 @@
 <div class="absolute top-5 left-5">
     !{var bookingShareType = booking.getShareRoomType();}
     @if(bookingShareType == ShareRoomType.YES)
-        <i class="fa-solid fa-users"></i>
+        <label class="label" aria-label="${context.localize("bookings.collaboration.share_room")}"
```

Comment:
This should be set either for aria-label or for title, not for both.
Also, having this be based on the name `YES` and symmetric to the other tooltips would be nicer.